### PR TITLE
Added Vision ZL7431

### PR DESCRIPTION
**Pull request by SkynetHome**
Added Vision ZL7431 to database per Issue #2817

### CODE REVIEW

**Comment by SkynetHome** [comment]
Sorry Chris I keep forgetting that I have to merge when I commit. I have got to get out of the habit of using github for commits.

File: bundles/binding/org.openhab.binding.zwave/database/vision/zl7431.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+				<Label lang="en">Set to 2 for a Single Wall Switch with Indicator
+					Light</Label>
+			</Item>
+		</Parameter>
+	</Configuration>
+	<Associations>
+		<Group>
+			<Index>1</Index>
+			<Maximum>5</Maximum>
+			<Label lang="en">Update Controller</Label>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/vision/zl7431.xml]: This should have the SetToController element

**Comment by SkynetHome** [comment]
Sorry getting sloppy! Fixing now still cant seem to merge commits. I committed new file on the Qubino and the only option I get is to amend previous commit, will try answering the question about replacing message differently this time maybe thats the cause of the commits not being merged.

**Comment by SkynetHome** [comment]
Most of the files in the database I come across with only 1 association group do not have the SetToController  line. Why is that?

**Comment by SkynetHome** [comment]
Just committed changes through Source Tree. Source Tree reflects that the file has been committed and I used Amend previous commit but I dont see a new update here. Is that because I did it right and didnt create a new commit or did the change not come through?

**Pull request closed without merging**